### ISSUE TITLE
cpp-extended-record-include-prefix is not always honoured 

### ISSUE THREAD

**Issue by milch** [dropbox/djinni#269]
Consider the following example: 

```
request = record +c {
    method : string;
    url : string;
    // etc... 
}

response = record +c {
    request : request;
    status_code : i32;
    // etc...
}
```

The generated code for `response_base.hpp` will be using `#include "request.hpp"` even when I use the flag `--cpp-extended-record-include-prefix "extended/"`.

**Comment by artwyman** [comment]
Confirmed this is a bug.  Seems to be specific to the case of an extended record being included in another record.

**Comment by artwyman** [comment]
Fixed in 6c869e1
